### DISCUSSION TITLE
Request for enabling windows authentication for WiltonDB BabelFish connection

### DISCUSSION THREAD

**Discussion by psrinivasa2** [babelfish-for-postgresql/babelfish-for-postgresql#28] [Q&A] [answered]
I am currently exploring the WiltonDB option and doing a POC around it. I understood from staticlibs (Alex Kasko) that **windows** **authentication** is NOT supported for WiltonDB BabelFish connection string. He also mentioned that change to enable windows authentication may be simple, but it requires more testing effort. We feel without windows authentication, our application may not be secure when integrating with WiltonDB. Though there were suggestion to encrypt the connection string password, we are not convinced with the security. **We would like to request WiltonDB team, If the windows authentication is enabled with testing effort, how much of time it will take and when it will be ready?** This is just to arrive our application release plan.

**Comment by staticlibs** [comment, marked as answer]
@psrinivasa2 , I will look into implementing SSPI support for TDS connections on Windows, but cannot give any guarantees at this point - this needs to be explored first. I've filed [#18](https://github.com/wiltondb/wiltondb/issues/18) to track this.

**Reply by psrinivasa2** [reply]
Thanks @staticlibs, please post us the progress on this.
